### DISCUSSION TITLE
Castle images

### DISCUSSION THREAD

**Discussion by ddugovic** [WandererXII/lishogi#372] [Ideas]
It could be a fun feature to show Shogi Wars-style popup images without resorting to OBS plugins like https://github.com/RossLitzenberger/KappaGen , although such a feature seems unlikely to be developed anytime soon.

"Very High Mino Castle"
https://twitch.tv/shogi_harbour/clip/ViscousElegantSnoodPeanutButterJellyTime

**Comment by WandererXII** [comment]
We can display pictures like this relatively easily - something like that is done in https://lishogi.org/learn#/17/1 with the repetition numbers. So what's left is detecting these positions and having copyright free designs to display.
This is also very gimmicky, which doesn't go well with the overall lichess/lishogi design. So if developed it would probably be off by default and having too many options isn't really a good thing... So I'm not sure
